### PR TITLE
add coverage related functions to global templates [CPCN-523]

### PR DESCRIPTION
**Pull request by devketanpro**
https://github.com/superdesk/newsroom-app-cp/pull/158

## Checklist

- [ ] This pull request is not adding new forms that use redux
- [ ] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence
- [ ] This pull request is replacing `lodash.get` with optional chaining for modified code segments

### CODE REVIEW

**Review by petrjasek — APPROVED** [review]

File: newsroom/agenda/utils.py [hunk 1]
```diff
@@ -164,6 +164,16 @@ def get_planning_coverages(item, plan_id):
     return [coverage for coverage in item.get("coverages") or [] if coverage.get("planning_id") == plan_id]
 
 
+def get_coverage_status(coverage: Dict[str, Any]) -> str:
+    coverage_status = coverage.get("coverage_status")
+    if coverage_status == "coverage intended":
+        return "Planned"
```
Comment (petrjasek) [inline newsroom/agenda/utils.py:170]: those would probably need translations

File: newsroom/agenda/utils.py [hunk 2]
```diff
@@ -164,6 +164,18 @@ def get_planning_coverages(item, plan_id):
     return [coverage for coverage in item.get("coverages") or [] if coverage.get("planning_id") == plan_id]
 
 
+def get_coverage_status(coverage: Dict[str, Any]) -> str:
+    coverage_status = coverage.get("coverage_status")
+    if coverage_status == "coverage intended":
+        return gettext("Planned")
+    elif coverage_status in ("coverage not decided yet", "coverage upon request"):
```
Comment (petrjasek) [inline newsroom/agenda/utils.py:171]: is there any requirement not to have specific status for `on request` coverages?

Comment (devketanpro) [reply newsroom/agenda/utils.py:171]: nope, in the templates, there is not any specific requirement for `on request` .

Comment (petrjasek) [reply newsroom/agenda/utils.py:171]: would keep those separate then

**Review by petrjasek — APPROVED** [review]

**Pull request merged**